src/main/scala/scorex/core/api/http/PeersApiRoute.scala

```diff
@@ -90,19 +92,25 @@ case class PeersApiRoute(peerManager: ActorRef,
 
 object PeersApiRoute {
 
-  case class PeerInfoResponse(address: String,
+  case class PeerInfoResponse(remoteAddress: String,
+                              localAddress: Option[String],
                               lastMessage: Long,
                               lastHandshake: Long,
                               name: String,
+                              agentName: String,
+                              protocolVersion: String,
                               connectionType: Option[String])
 
   object PeerInfoResponse {
 
     def fromAddressAndInfo(address: InetSocketAddress, peerInfo: PeerInfo): PeerInfoResponse = PeerInfoResponse(
       address.toString,
+      None,
       0,
       peerInfo.lastHandshake,
       peerInfo.peerSpec.nodeName,
+      peerInfo.peerSpec.agentName,
+      peerInfo.peerSpec.protocolVersion.toString,
```

Comment:
do we have agentName and protocol version for non-connected peers? See the method `allPeers` above.

Reply:
Yes, we do; although when a peer is added from the config file we still don't know those data, so they are hard coded as agentName -> `unknown` and protocolVersion -> `Version.initial`. These data will be updated once we establish a connection

Reply:
ok, clear